include/QuantumComputation.hpp

```diff
@@ -607,6 +618,19 @@ namespace qc {
             checkQubitRange(qubit);
             emplace_back<NonUnitaryOperation>(getNqubits(), qubit, clbit);
         }
+
+        void measure(dd::Qubit qubit, std::pair<std::string, std::size_t> clbit) {
```

Comment:
A typdef for Bit already exists:
`using Bit   = std::uint64_t;`
Since `std::pair<std::string, std::size_t>` is only used once in this specific context, I feel like creating a typdef is not justified.